modules/contact/test/tests/mortar_aux_kernels/block-dynamics-aux-fretting-wear-test-action.i

```diff
@@ -0,0 +1,283 @@
+starting_point = 0.5e-1
+offset = -0.045
+
+[GlobalParams]
+  displacements = 'disp_x disp_y'
+[]
+
+[Mesh]
+  [file]
+    type = FileMeshGenerator
+    file = long-bottom-block-1elem-blocks.e
+  []
+[]
+
+[Variables]
+  [disp_x]
+    block = '1 2'
+  []
+  [disp_y]
+    block = '1 2'
+  []
+[]
+
+[Problem]
+  material_coverage_check = false
+  kernel_coverage_check = false
+[]
+
+[ICs]
+  [disp_y]
+    block = 2
+    variable = disp_y
+    value = '${fparse starting_point + offset}'
+    type = ConstantIC
+  []
+[]
+
+[Kernels]
+  [DynamicTensorMechanics]
```

Comment:
It works. I'll be using it in an upcoming commit, but needed to regold because found differences on the order of 1.0e-3 which can be attributed to the execution of the auxiliary kernels for velocity and acceleration.